qiskit/optimization/converters/ising_to_quadratic_program.py

```diff
@@ -77,7 +83,7 @@ def encode(self, qubit_op: WeightedPauliOperator, offset: float = 0.0) -> Quadra
                 # The coefficient of the quadratic term in `QuadraticProgram` is
                 # 4 * weight of the pauli
                 coef = weight * 4
-                quadratic[(i, j)] = coef
+                quadratic_terms[(i, j)] = coef
```

Comment:
No need to use `()`. `quadratic_terms[i, j]` should work.

Reply:
Alright, I fixed it.